src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationRow.jsx

```diff
@@ -0,0 +1,183 @@
+            <SentryAppName>
+              {showPublishStatus ? (
+                <SentryAppLink to={`/settings/${orgId}/developer-settings/${app.slug}/`}>
+                  {app.name}
+                </SentryAppLink>
+              ) : (
+                app.name
+              )}
+            </SentryAppName>
+            <SentryAppDetails>
+              {showPublishStatus ? (
+                <PublishStatus published={app.status === 'published'} />
+              ) : (
+                <React.Fragment>
+                  <Status enabled={isInstalled} />
+                  <StyledLink onClick={() => {}}>Learn More</StyledLink>
+                </React.Fragment>
+              )}
+            </SentryAppDetails>
+          </SentryAppBox>
+          {!showPublishStatus ? (
+            <Box>
+              {!isInstalled ? (
+                <Button
+                  onClick={() => this.props.onInstall(app)}
+                  size="small"
+                  icon="icon-circle-add"
+                  className="btn btn-default"
+                >
+                  {t('Install')}
+                </Button>
+              ) : (
+                this.renderUninstall(installs[0])
+              )}
+            </Box>
+          ) : (
+            <Box>
+              <Button
+                icon="icon-trash"
+                size="small"
+                onClick={() => {}}
+                className="btn btn-default"
+              />
+            </Box>
+          )}
+        </StyledFlex>
+      </SentryAppItem>
+    );
+  }
+}
+
+const SentryAppItem = styled(PanelItem)`
+  flex-direction: column;
+  padding: 5px;
+`;
+
+const StyledFlex = styled(Flex)`
+  justify-content: center;
+  padding: 10px;
+`;
+
+const SentryAppBox = styled(Box)`
+  padding-left: 15px;
+  padding-right: 15px;
+  flex: 1;
+`;
+
+const SentryAppDetails = styled(Flex)`
+  align-items: center;
+  margin-top: 6px;
+  font-size: 0.8em;
+`;
+
+const SentryAppName = styled('div')`
+  font-weight: bold;
+`;
+
+const StyledLink = styled(Link)`
+  color: ${p => p.theme.gray2};
+`;
+
+const SentryAppLink = styled(Link)`
+  color: ${props => props.theme.textColor};
+`;
+
+const StyledButton = styled(Button)`
+  color: ${p => p.theme.gray2};
+`;
+
+const Status = styled(
+  withTheme(({enabled, ...props}) => {
+    return (
+      <Flex align="center">
+        <CircleIndicator
+          size={6}
+          color={enabled ? props.theme.success : props.theme.gray2}
+        />
+        <div {...props}>{enabled ? t('Installed') : t('Not Installed')}</div>
+      </Flex>
+    );
+  })
+)`
+  color: ${props => (props.enabled ? props.theme.success : props.theme.gray2)};
+  margin-left: ${space(0.5)};
+  font-weight: light;
+  &:after {
+    content: '|';
+    color: ${p => p.theme.gray1};
+    margin-left: ${space(0.75)};
+    font-weight: normal;
+  }
+  margin-right: ${space(0.75)};
+`;
+
+const PublishStatus = styled(
+  withTheme(({published, ...props}) => {
```

Comment:
I use it for the color distinction b/n `published` and `unpublished`

Reply:
I'm pretty sure you don't need the `withTheme` wrapping the element though since you're not using the theme inside the element itself right, only in the CSS.